tests/wpunit/Tribe/ContextTest.php

```diff
@@ -1623,8 +1620,11 @@ public function should_not_change_values_after_repopulating_with_cache() {
 		tribe_update_option( '__after_repopulate__', '__value_after_repopulate__' );
 
 		$context = new Context;
+		// This is to ensure the static properties are reset so the test before does not affect the test after.
+		$context->dangerously_reset_static_properties();
```

Comment:
It looks like this, being a call to a static method, should be moved to a `@before/@after` method to avoid leaking state between test methods in this class and other test cases.